common/scala/src/main/scala/org/apache/openwhisk/http/PoolingRestClient.scala

```diff
@@ -72,16 +76,19 @@ class PoolingRestClient(
   // Additional queue in case all connections are busy. Should hardly ever be
   // filled in practice but can be useful, e.g., in tests starting many
   // asynchronous requests in a very short period of time.
-  private val requestQueue = Source
-    .queue(queueSize, OverflowStrategy.dropNew)
```

Comment:
If the overflow strategy is not specified, what happens now in case the queue is full?